pkgs/Enter-tainer/typstyle/registry.yaml

```diff
@@ -0,0 +1,15 @@
+packages:
+  - type: github_release
+    repo_owner: Enter-tainer
+    repo_name: typstyle
+    description: Beautiful and reliable typst code formatter
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: "true"
+        asset: typstyle-{{.OS}}-{{.Arch}}
+        format: raw
+        replacements:
+          amd64: x64
+        supported_envs:
+          - linux
+          - darwin
```

Comment:
At least, the latest version supports windows.
https://github.com/Enter-tainer/typstyle/blob/29ca37c4b4cea2d8ecf59235170e61b8dc59642e/.github/workflows/release.yml#L68-L75
https://github.com/Enter-tainer/typstyle/releases/tag/v0.11.26

Reply:
Hmm, scaffold command seems not to recognize `win32` as windows binary.

Reply:
Yes. scaffold isn't perfect. It can't be helped. So we need to fix it manually.